test/sanity/validate-modules/main.py

```diff
@@ -1301,6 +1301,14 @@ def _validate_argument_spec(self, docs, spec, kwargs):
                          "but documentation defines choices as (%r)" % (arg, arg_choices, doc_choices))
                 )
 
+        for arg in args_from_argspec:
+            if not str(arg).isidentifier():
```

Comment:
This will traceback on Python 2.x since `str.isidentifier` was added in Python 3.x. Do we want to make `validate-modules` Python 3.x only going forward?

Reply:
I was under the impression we were only supporting it on py3 already.  I've closed a few issues about py2 compatibility.

So yes, I'd prefer just targeting py3.

Reply:
In that can you update the sanity test to reflect that?

https://github.com/ansible/ansible/blob/d901ba18d11063bd95843c0fbf8e6ec85b609663/test/runner/lib/sanity/validate_modules.py#L42

Here's how it's done for the `pylint` test:

https://github.com/ansible/ansible/blob/d901ba18d11063bd95843c0fbf8e6ec85b609663/test/runner/lib/sanity/pylint.py#L65-L67

Reply:
Updated